Remove Chat log to improve download times

Preloading opened the pull request:
btw not done

Preloading commented:
also i love your videos, will this be turned into a video? I am a first time twitch chatter, and i love the coding, please post meme generator and https://flexboxfroggy.com/ it very easy

tizu69 reviewed (approved):
yea I'm not sure if that huge gitignore is faster to download than the chat history.. but whatever, lgtm

benny-yamagata commented:
Saying you don't need to review just hit accept merge request is just a red flag to review even more thoroughly. 

I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.

fily-gif commented:
> Saying you don't need to review just hit accept merge request is just a red flag to review even more thoroughly.
> 
> I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.

LGTM, they just pasted a .gitignore template and added the chat log

Preloading commented:
> Saying you don't need to review just hit accept merge request is just a red flag to review even more thoroughly. 
> 
> I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.

Yes I am aware, I am saying that you  don't need to click the few approve this code buttons, right before merging, not that you don't need to review the code, to see that it is safe, etc before you merge.

Preloading commented:
> > Saying you don't need to review just hit accept merge request is just a red flag to review even more thoroughly.
> > 
> > I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.
> 
> LGTM, they just pasted a .gitignore template and added the chat log

I added the python template, removed the chat log file, and added the chat log to the gitignore, before more PRs were merged, already doing the stuff. It's small, but isn't this Doug's first time using GitHub? He wouldn't figure out gitignore for a while. It was mainly intended to prevent the chat log, which could potentially contain stuff that you don't want on a public repo. Download times, was because every single commit, you would need to redownload the chat log, and they could get long fast

Preloading commented:
I have also resolved the merge conflict

Preloading commented:
And changed the title to be more reflective

FifthTundraG reviewed (approved):
User-generated logs like ChatHistoryBackup.txt shouldn't be included in an open-source repo, anyway.

Preloading commented:
> User-generated logs like ChatHistoryBackup.txt shouldn't be included in an open-source repo, anyway.

yes... thats what this was suppose to solve

benny-yamagata commented:
> > Saying you don't need to review just hit accept merge request is just a red flag to review even more thoroughly. 
> > 
> > I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.
> 
> Yes I am aware, I am saying that you  don't need to click the few approve this code buttons, right before merging, not that you don't need to review the code, to see that it is safe, etc before you merge.

My bad, misunderstood the intent!

StealthBadger747 reviewed (approved):
Basic stuff. LGTM

aprilthepink commented:
> I would hope that anyone who is maintaining open source code would review any merge requests, especially when they know that code may be consumed by thousands of individuals.

so much this!
